docker/local/start.sh

```diff
@@ -3,8 +3,11 @@
 # Compile translations once during start
 python manage.py compilemessages
 
+# Run Django development server to serve static files
+python manage.py collectstatic --noinput
+
 while true; do
-    echo "Re-starting Django runserver_plus!"
-    python manage.py runserver_plus 0.0.0.0:8000
+    echo "Starting Uvicorn ASGI server!"
+    uvicorn config.asgi:application --host 0.0.0.0 --port 8000
```

Comment:
Just to be sure: It is actually necessary for the uviconr command to be inside the `while` loop?